dask_planner/src/sql.rs

```diff
@@ -152,6 +152,14 @@ impl ContextProvider for DaskSQLContext {
                 let rtf: ReturnTypeFunction = Arc::new(|_| Ok(Arc::new(DataType::Int64)));
                 return Some(Arc::new(ScalarUDF::new(name, &sig, &rtf, &fun)));
             }
+            "dsql_totimestamp" => {
+                let sig = Signature::variadic(
+                    vec![DataType::Date64, DataType::Utf8],
+                    Volatility::Immutable,
+                );
```

Comment:
We'll probably want this to be either `Signature::one_of` if we want to allow a variety of numeric types as input; cc @andygrove in case you know of an intuitive way to write out this function signature as it seems like it might be cumbersome to enumerate all the different supportable numeric types here

Reply:
No, those should be fine 🙂 I was mostly thinking about if there was a simpler way to enumerate all the variations of ints, i.e. `[U]Int[8|16|32|64]`